### PR TITLE
Contract changes for session and create user

### PR DESCRIPTION
**Pull request by ganesanarun**

### CODE REVIEW

File: src/main/resources/static/swagger.yaml [hunk 1]
```diff
@@ -43,6 +43,70 @@ paths:
...
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UserCreationRequest'
+      responses:
+        '200':
+          description: Successful creation of consent artefact.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PatientSearchRepresentation'
```
Comment (angshu) [inline src/main/resources/static/swagger.yaml]: this ought to return user representation, right?

File: src/main/resources/static/swagger.yaml [hunk 2]
```diff
@@ -440,4 +504,30 @@ components:
           example: Curve25519/32byte random key
         keyValue:
           type: string
-          format: string
\ No newline at end of file
+          format: string
+    SessionRequest:
+      type: object
+      required:
```
Comment (angshu) [inline src/main/resources/static/swagger.yaml:505]: Should also take in "display".  We should use this "display" element to show the Doctor's or any Users name.

**Review by angshu — DISMISSED** [review]

File: src/main/resources/static/swagger.yaml (hunk 2 above)
Comment (ganesanarun) [reply src/main/resources/static/swagger.yaml:505]: I think it's okay to show username as display name right now. It's mock UI we are developing.

File: src/main/resources/static/swagger.yaml (hunk 1 above)
Comment (ganesanarun) [reply src/main/resources/static/swagger.yaml]: We don't return anything, as it's an API we created for creating new doctor login.

**Review by Apoorva-GA — APPROVED** [review]

**Pull request merged**